src/chapter_1_crates/section_2_http.md

```diff
@@ -7,6 +7,45 @@ but it can be changed to use NativeTLS which uses the TLS native to the platform
 
 Ratelimiting is included out-of-the-box, along with support for proxies.
 
+## Features
+
+### Deserialization
+
+`twilight-gateway` supports [`serde_json`] and [`simd-json`] for deserializing
+and serializing events.
+
+#### Simd JSON
```

Comment:
I think we should clarify the capitlization of SIMD here and the other mentions of it, I don't know what the right one is but the website uses all lowercase https://simdjson.org/